public/static/locales/th/manage.json

```diff
@@ -48,7 +48,7 @@
   },
   "classrooms": {
     "title": "ห้องเรียน",
-    "desc": "นักเรียนได้รับการจัดกลุ่มเป็นห้องเรียนต่าง ๆ ทุกปี พิมพ์ใบรายชื่อของทุกห้องเรียนพร้อมกันได้ที่นี่",
+    "desc": "นักเรียนได้รับการจัดกลุ่มเป็นห้องเรียนต่าง ๆ ทุกปี พิมพ์ใบรายชื่อของทุกห้องเรียนพร้อมกันได้ที่นี่",
```

Comment:
How is this different!? lol

Reply:
Mai yamok must be proceeded by a non-breaking space.